workflow-beta2/tpl/deis-logger-fluentd-daemon.yaml

```diff
@@ -0,0 +1,40 @@
+#helm:generate helm tpl -d $HELM_GENERATE_DIR/tpl/generate_params.toml -o $HELM_GENERATE_DIR/manifest/deis-logger-fluentd-daemon.yaml $HELM_GENERATE_DIR/tpl/deis-logger-fluentd-daemon.yaml
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: deis-logger-fluentd
+  namespace: deis
+  labels:
+    heritage: deis
+spec:
```

Comment:
@jchauncey that's the entire `spec`? This is copied from https://github.com/deis/charts/blob/master/workflow-dev/manifests/deis-logger-fluentd-daemon.yaml#L19-L44